probables/cuckoo/countingcuckoo.py

```diff
@@ -66,6 +68,60 @@ def __init__(
             hash_function,
         )
 
+    @classmethod
+    def init_error_rate(
+        cls,
+        error_rate: float,
+        capacity: int = 10000,
+        bucket_size: int = 4,
+        max_swaps: int = 500,
+        expansion_rate: int = 2,
+        auto_expand: bool = True,
+        hash_function: typing.Optional[SimpleHashT] = None,
+    ):
+        """Initialize a simple Cuckoo Filter based on error rate
+
+        Args:
+            error_rate (float):
+            capacity (int): The number of bins
+            bucket_size (int): The number of buckets per bin
+            max_swaps (int): The number of cuckoo swaps before stopping
+            expansion_rate (int): The rate at which to expand
+            auto_expand (bool): If the filter should automatically expand
+            hash_function (function): Hashing strategy function to use \
+            `hf(key)`
+        Returns:
+            CuckooFilter: A Cuckoo Filter object"""
+        cku = CountingCuckooFilter(
+            capacity=capacity,
+            bucket_size=bucket_size,
+            auto_expand=auto_expand,
+            max_swaps=max_swaps,
+            expansion_rate=expansion_rate,
+            hash_function=hash_function,
+        )
+        # reset fingerprint based on error rate
+        cku._fingerprint_size = int(math.ceil(math.log(1.0 / error_rate, 2) + math.log(2 * cku.bucket_size, 2)))
```

Comment:
Isn't it a dupe?

Reply:
technically, no, it isn't since this is in bits where the value passed in is in bytes. So when force an error rate, then the fingerprint size in bits may not be an exact multiple of 8. 

I moved this to a function for the two places that we need to calculate the fingerprint size.